README.md

```diff
@@ -64,23 +80,30 @@ beforehand.  This is done by letting npm put the dependencies into a
 converting that into a manifest snipped called
 `generated-sources.json`.
 
-You need to generate the `package-lock.json` file from the
-deltachat-desktop release you need.  Go to the git repository,
-chechout the correct tag and run (e.g. using docker):
+Upstream ships the package-lock.json file so it should not be
+necessary to generate it.  However, sometimes the file is not
+updated in lockstep with package.json and then dependencies will
+be missing during build time.
```

Comment:
Is this why that file is still in this repository?

Reply:
yeah. in the manifest is a reference to the bug report.